tool.go

```diff
@@ -498,7 +498,21 @@ func main() {
 	cmdServe.Run = func(cmd *cobra.Command, args []string) {
 		options.BuildTags = strings.Fields(*tags)
 		dirs := append(filepath.SplitList(build.Default.GOPATH), build.Default.GOROOT)
-		sourceFiles := http.FileServer(serveCommandFileSystem{options: options, dirs: dirs, sourceMaps: make(map[string][]byte)})
+		root := ""
```

Comment:
That's a good point about emphasizing that the exact value is unimportant. I'll make that change. Thanks for the feedback :D